src/app/application.cpp

```diff
@@ -204,6 +208,24 @@ const QBtCommandLineParameters &Application::commandLineArgs() const
     return m_commandLineArgs;
 }
 
+#ifdef Q_OS_WIN
+int Application::memoryWorkingSetLimit() const
+{
+    return std::max(m_storeMemoryWorkingSetLimit.get(), 0);
```

Comment:
`m_storeMemoryWorkingSetLimit.get(<default value>)`
I would even remove `std::max()` and let the system call deal with invalid parameters.